IdentityCore/src/requests/broker/MSIDBrokerKeyProvider.m

```diff
@@ -209,4 +209,68 @@ - (BOOL)deleteSymmetricKeyWithError:(NSError **)error
     return YES;
 }
 
+- (BOOL)saveApplicationToken:(NSString *)appToken forClientId:(NSString *)clientId error:(NSError **)error
+{
+    NSString *tag = [NSString stringWithFormat:@"%@-%@", MSID_BROKER_APPLICATION_TOKEN_TAG, clientId];
```

Comment:
nit: add info level log that we're saving a token?

Reply:
Addressed.